src/tests/Common/Directory.Build.targets

```diff
@@ -171,34 +171,15 @@
       <RunTimeDependencyCopyLocal Include="$(CoreDisToolsLibrary)" Condition="$(CopyCoreDisToolsToCoreRoot)" />
     </ItemGroup>
 
-<<<<<<< HEAD
-    <PropertyGroup>
-      <IlcDir>$(CoreCLRArtifactsPath)ilc</IlcDir>
-    </PropertyGroup>
-
-    <!-- When cross-compiling tests (such as is the case for Wasm), we depends on the host's ILC being built. -->
-    <!-- This is a workaround for ILC's project not supporting cross-targeting. Remove once that's implemented. -->
-    <PropertyGroup Condition="'$(TargetArchitecture)' != '$(BuildArchitecture)'">
-      <IlcDir>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'artifacts', 'bin', 'coreclr', '$(BuildOS).$(BuildArchitecture).$(CoreCLRConfiguration)'))ilc</IlcDir>
-    </PropertyGroup>
-
-    <Error Condition="!Exists('$(IlcDir)')"
-           Text="'$(IlcDir)' does not exist. Subsets clr.jit, clr.wasmjit and nativeaot.ilc for $(BuildOS) $(BuildArchitecture) in $(CoreCLRConfiguration) configuration must be built before building this project." />
```

Comment:
My preference would be to move `HostCoreCLRArtifactsPath` definition to here, retain the error and the TODO. It is still relevant (though no cross-build for ILC exists even in current runtime `main`).